### PR TITLE
[Merged by Bors] - chore(IsAlgClosed/Classification): make theorems more universe polymorphic

### PR DESCRIPTION
**Pull request by ChrisHughes24**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/FieldTheory/IsAlgClosed/Classification.lean [hunk 1]
```diff
@@ -73,76 +73,94 @@ end Classification
...
 
-variable {R K : Type u} [CommRing R] [Field K] [Algebra R K] [IsAlgClosed K]
-variable {ι : Type u} (v : ι → K)
+variable {R : Type u} {K : Type v} [CommRing R] [Field K] [Algebra R K] [IsAlgClosed K]
+variable {ι : Type w} (v : ι → K)
 
 theorem cardinal_le_max_transcendence_basis (hv : IsTranscendenceBasis R v) :
-    #K ≤ max (max #R #ι) ℵ₀ :=
+    Cardinal.lift.{max u w} #K ≤ max (max (Cardinal.lift.{max v w} #R)
+      (Cardinal.lift.{max u v} #ι)) ℵ₀ :=
```
Comment (jcommelin) [inline Mathlib/FieldTheory/IsAlgClosed/Classification.lean:81]: I think it's worth preserving the old statements for convenience and readability. Could you please add them as corollaries?

**Comment by jcommelin** [comment]
And my apologies for the late review. I don't know how this went under the radar for almost 2 months.

**Comment by ChrisHughes24** [comment]
> And my apologies for the late review. I don't know how this went under the radar for almost 2 months.

It's because it didn't build for the last two months and I fixed it yesterday.

File: Mathlib/FieldTheory/IsAlgClosed/Classification.lean (hunk 1 above)
Comment (ChrisHughes24) [reply Mathlib/FieldTheory/IsAlgClosed/Classification.lean:90]: done

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**